happiness-backend/api/models/schema.py

```diff
@@ -223,3 +225,22 @@ class ReadsSchema(ma.Schema):
     happiness_id = ma.Int(dump_only=True, required=True)
     user_id = ma.Int(dump_only=True, required=True)
     timestamp = ma.Str(dump_only=True, required=True)
+
+
+class HappinessRangeSchema(ma.Schema):
+    id = ma.Int()
+    page = ma.Int()
+    count = ma.Int()
+    low = ma.Int()
+    high = ma.Int()
+
+
+class HappinessMultiFilterSchema(ma.Schema):
+    user_id = ma.Int()
+    page = ma.Int()
+    count = ma.Int()
+    low = ma.Int(missing=11)
+    high = ma.Int(missing=11)
+    start = ma.Str(missing=None)
+    end = ma.Str(missing=None)
```

Comment:
Also look into refactoring this to use `ma.Date()` I saw it in Jonathan's code so feel free to ask him.